src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -43,7 +44,13 @@ public CommandBox(Logic logic) {
         this.commandList = logic.getCommandList();
         // calls #setStyleToDefault() whenever there is a change to the text of the command box.
         commandTextField.textProperty().addListener((unused1, unused2, unused3) -> setStyleToDefault());
-        TextFields.bindAutoCompletion(commandTextField, commandList);
+        //@@ChenSongJian
```

Comment:
The proper way to do author tags in this module (required by the collate tool sadly) is
```
//@@author YourName
public void yourFunction() {
}
//@@author
public void noonesFunction() {
}